content/en/docs/tasks/access-application-cluster/connecting-frontend-backend.md

```diff
@@ -87,50 +89,59 @@ Events:
 
 ## Creating the backend Service object
 
-The key to connecting a frontend to a backend is the backend
+The key to sending requests from a frontend to a backend is the backend
 Service. A Service creates a persistent IP address and DNS name entry
 so that the backend microservice can always be reached. A Service uses
 {{< glossary_tooltip text="selectors" term_id="selector" >}} to find
 the Pods that it routes traffic to.
 
 First, explore the Service configuration file:
 
-{{< codenew file="service/access/hello-service.yaml" >}}
+{{< codenew file="service/access/backend-service.yaml" >}}
 
-In the configuration file, you can see that the Service routes traffic to Pods
-that have the labels `app: hello` and `tier: backend`.
+In the configuration file, you can see that the Service, named `backend` routes 
+traffic to Pods that have the labels `app: hello` and `tier: backend`.
 
-Create the `hello` Service:
+Create the `backend` Service:
 
 ```shell
-kubectl apply -f https://k8s.io/examples/service/access/hello-service.yaml
+kubectl apply -f https://k8s.io/examples/service/access/backend-service.yaml
 ```
 
-At this point, you have a backend Deployment running, and you have a
-Service that can route traffic to it.
+At this point, you have a `backend` Deployment running seven replicas of your `hello`
```

Comment:
Sounds to me that 7 replicas is an overkill. Can/shall we reduce it to 3?

Reply:
Agreed, reduced to 3 (here as well as deployment manifest and sample apply output)